jenkins/pr-multi-build/Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+    agent { label 'docker' }
+    parameters {
+        string(name: "NUM_RUNS", defaultValue: "3",
+            description: "The (integer) number of times to run the automation tests.")
+    }
+
+    stages {
+        stage('Run tests') {
+            parallel {
+                stage('Run tests in parallel') {
+                    steps {
+                        script {
+                            def tests = [:]
+
+                            def numRuns = Integer.parseInt("$NUM_RUNS")
+
+                            for (int i = 0; i < numRuns; i++) {
+                                def params = [
+                                    string(name: "RUN_ID", value: "$i"),
```

Comment:
The swatomation job doesn't actually use this, but a unique parameter list is needed to ensure the plugin triggers separate builds